src/hooks/useFormWorkspaceSize.ts

```diff
@@ -0,0 +1,51 @@
+import { useLayoutEffect, useMemo, useState } from 'react';
+import { pxToRem } from '../utils/common-utils';
+
+/**
+ * The width of the supported workspace variants in rem
+ */
+const narrowWorkspaceWidth = 26.25;
+const widerWorkspaceWidth = 32.25;
+const extraWideWorkspaceWidth = 48.25;
+
+type WorkspaceSize = 'narrow' | 'wider' | 'extra-wide' | 'ultra-wide';
```

Comment:
This indicates that `extra-wide` and `ultra-wide` might be doing the same thing so maybe we should drop `extra-wide` naming